lib/aio/github.py

```diff
@@ -133,17 +135,11 @@ async def open_issue(self, repo: str, issue: JsonObject) -> None:
         await self.post(f'repos/{repo}/issues', issue)
 
     async def read_file(self, subject: Subject, filename: str) -> str | None:
-        async def read_once() -> str | None:
-            try:
-                async with self.session.get(self.content / subject.repo / subject.sha / filename) as response:
-                    logger.debug('response %r', response)
-                    return await response.text()
-            except aiohttp.ClientResponseError as exc:
-                if exc.status == 404:
-                    return None
-                raise
-
-        return await retry(read_once)
+        content = await self.get_obj(f'repos/{subject.repo}/contents/{filename}', {'ref': subject.sha})
+        try:
+            return base64.b64decode(str(content['content'])).decode().strip()
```

Comment:
Can you jsonutils this a bit?

Reply:
There is no get_bytes tho so that seems unlikely?

Reply:
Everything is also heavily string based so changing this is a bit more work, ie. the default argument for `_get` is a string etc.

Reply:
I misunderstood my code, honestly get_str() makes it kinda worse but here it is.